src/components/Menu/LanguageNavigation/LanguageNavigation.tsx

```diff
@@ -7,12 +7,12 @@ import {
   ReactSelectOption,
   Select,
 } from 'src/components';
-import { PageLanguageContext } from 'src/contexts';
 
 import { DEFAULT_LANGUAGE, DEFAULT_PREFERRED_LANGUAGE, SDK_INTERFACES } from '../../../../data/createPages/constants';
-import { cacheVisitPreferredLanguage } from 'src/utilities';
 import { dropdownContainer, horizontalNav } from './LanguageNavigation.module.css';
 import SDKInterfacePanel from '../../SDKInterfacePanel/SDKInterfacePanel';
+import { usePageLanguage } from 'src/contexts';
+import { navigate } from 'gatsby';
```

Comment:
Looking at the general conventions around, this import should probably be moved further up to sit alongside the other third party imports (i.e. L3)